src/runtime-rs/crates/resource/src/cgroups/utils.rs

```diff
@@ -14,3 +25,19 @@
 pub(crate) fn gen_overhead_path(path: &str) -> String {
     format!("kata_overhead/{}", path.trim_start_matches('/'))
 }
+
+pub(crate) fn rename_cgroup_path(path: &str) -> String {
+    if path.is_empty() {
+        return DEFAULT_RESOURCE_CONTROLLER_ID.to_string();
+    }
+
+    let cgroup_path = Path::new(path);
+    let cgroup_path_dir = cgroup_path.parent().unwrap().to_string_lossy().to_string();
+    let cgroup_path_name = format!(
+        "{}_{}",
+        CGROUP_KATA_PREFIX,
+        cgroup_path.file_name().unwrap().to_str().unwrap()
```

Comment:
if cgroup_path is '/', this sentence would be panic

Reply:
fixed, and I added some UT.